release-notes/Cabal-3.12.1.0.md

```diff
@@ -0,0 +1,54 @@
+Cabal and Cabal-syntax 3.12.1.0 changelog and release notes
+---
+
+
+### Significant changes
+
+- Update GHC arguments normalization and GHC options rendering [#9729](https://github.com/haskell/cabal/issues/9729) [#10014](https://github.com/haskell/cabal/pull/10014)
+
+  The flags `-fdiagnostics-as-json`, `-fprint-error-index-lists`, `-fbreak-points`, `-dipe-stats`, `-ffamily-application-cache`, `-fprint-redundant-promotion-ticks`, `-fshow-error-context` and `-funoptimized-core-for-interpreter` have been added to the flags that do not cause recompilation.
+
+- Warn on missing `default-language` [#9620](https://github.com/haskell/cabal/issues/9620) [#9766](https://github.com/haskell/cabal/pull/9766)
+
+  - To help the adoption of GHC language editions, `cabal check` will now
+    warn about missing `default-language`.
+
+- Add mhs as a known Haskell compiler [#9878](https://github.com/haskell/cabal/pull/9878)
```

Comment:
Minor: MHS

Reply:
Better yet, spell it out too, as it is more commonly known as MicroHS. And maybe link it to https://github.com/augustss/MicroHs

Reply:
Agree.